lib/ProxySQL_GloVars.cpp

```diff
@@ -75,9 +75,9 @@ ProxySQL_GlobalVariables::ProxySQL_GlobalVariables() {
 	opt->add((const char *)"",0,0,0,(const char *)"Starts only the admin service",(const char *)"-n",(const char *)"--no-start");
 	opt->add((const char *)"",0,0,0,(const char *)"Run in foreground",(const char *)"-f",(const char *)"--foreground");
 	opt->add((const char *)"",0,0,0,(const char *)"Do not restart ProxySQL if crashes",(const char *)"-e",(const char *)"--exit-on-error");
-	opt->add((const char *)"~/proxysql.cnf",0,1,0,(const char *)"Configuraton file",(const char *)"-c",(const char *)"--config");
+	opt->add((const char *)"/etc/proxysql.cnf",0,1,0,(const char *)"Configuraton file",(const char *)"-c",(const char *)"--config");
 	//opt->add((const char *)"",0,0,0,(const char *)"Enable custom memory allocator",(const char *)"-m",(const char *)"--custom-memory");
-	opt->add((const char *)"",0,1,0,(const char *)"Datadir",(const char *)"-D",(const char *)"--datadir");
+	opt->add((const char *)"/var/lib/proxysql",0,1,0,(const char *)"Datadir",(const char *)"-D",(const char *)"--datadir");
```

Comment:
The default is empty so that by default it uses the current directory as datadir.
Also in this case the idea is to try new versions or new configurations without affecting the system datadir.
That is why the [init script](https://github.com/sysown/proxysql/blob/v1.2.0/etc/init.d/proxysql#L17) explicitly set config file and datadir .